nix/shell.nix

```diff
@@ -1,5 +1,6 @@
-{ config ? {}
-, pkgs ? import ./pkgs.nix { inherit config; } }:
+{
+  pkgs ? import <nixpkgs> { }
```

Comment:
https://nix.dev/guides/best-practices#reproducible-nixpkgs-configuration

Reply:
Yeah, we don't need `<nixpkgs>` as a fallback.
Not sure it will even work with flakes.

Reply:
It will probably fail unless `--impure` is used unless `pkgs` is provided from the flake.